### ISSUE TITLE
About Snow100K dataset link

### ISSUE THREAD

**Issue by WUKUNTAI-0211** [sunshangquan/Histoformer#2]
Dear @sunshangquan ,

Thanks for your impressive work on adverse weather conditions.

I found that the download link for the Snow100K dataset in the paper is no longer valid. Could you please provide an alternative download link?

I am looking forward to hearing from you soon!

Best regards,

**Comment by sunshangquan** [comment]
Hi @Jerry-Jimmy , you could refer to [[Google Drive]](https://drive.google.com/file/d/1tfeBnjZX1wIhIFPl6HOzzOKOyo0GdGHl/view?usp=sharing) now for the training set and I will upload the test sets tomorrow.

**Comment by sunshangquan** [comment]
Hi @Jerry-Jimmy , the test sets of Snow100K-S, -L and realistic are uploaded on [[Google Drive]](https://drive.google.com/drive/folders/1kLPWAev5f6OutKFALhxthuO1XCF5U1Yu?usp=drive_link) or [[Baidu disk]](https://pan.baidu.com/s/1oiL0IrB1vWo902G-jWk8LA?pwd=yuia) with pin: ```yuia```.

**Comment by ducnt1210** [comment]
Dear @sunshangquan, by any chance do you have the snow masks from the dataset as well? Thank you.

**Comment by sunshangquan** [comment]
Hi @ducnt1210 , I can find mask folders for Snow100K-S (753MB), -L (1.2GB), and -M (845MB), all in .jpg format. Which one do you need?

**Comment by ducnt1210** [comment]
Thank you for your quick response, @sunshangquan. I would greatly appreciate it if you could provide all three mask folders. Thank you.

**Comment by sunshangquan** [comment]
Do you mind Baiduyun? Because they are huge but not needed by most people.

**Comment by ducnt1210** [comment]
Unfortunately, I have difficulty accessing Baidu. Would it be possible for you to upload the Snow100K-M masks to Google Drive instead? Just the Snow100K-M folder would be incredibly helpful. Thank you.

**Comment by sunshangquan** [comment]
Okay, I have uploaded them on [[Google Drive]](https://drive.google.com/drive/folders/11kGq5VB6hyEx-GTONvH9Z4cH7EhpjaJ_?usp=sharing) and [[BaiduYun]](https://pan.baidu.com/s/1Il24NXXt748nsL2nLpXI3g?pwd=hstm) [pin: hstm]. The Drive link may be unavailable one day because my storage is not large enough.

**Comment by ducnt1210** [comment]
Thank you so much, @sunshangquan! I really appreciate your time and effort. Thank you again.

**Comment by sunshangquan** [comment]
Hi @ducnt1210 , Glad to help you! If possible, I will appreciate it if you could cite our work~ Thank you for your attention.

**Comment by Ly403** [comment]
Hi @sunshangquan,

First, thank you for making the Snow100K dataset available. It's a great resource.

I've been using the training data from the Google Drive link, and it looks like it's a smaller version of the original dataset. I'm interested in working with the full 50,000-image training set.

Would you happen to have a download link for the complete dataset? Any information you can provide would be a great help.

Thanks again!

**Comment by 12345678901234567800001882277777** [comment]
Dear @sunshangquan ,

I am currently working on a project related to snow removal in images, and I would like to use the Snow100K dataset for training. I noticed that the download link provided on GitHub contains a mixture of all weather conditions, which makes it difficult to extract the Snow100K training subset.

Would it be possible for you to provide a separate download link for the 50,000 Snow100K training images? I would be extremely grateful for your help.

Thank you very much for your time and support！

**Comment by logoflog** [comment]
Dear @sunshangquan ,
请问您可以给一份snow100K-M的训练集和测试集吗，麻烦您了

**Comment by SeWngDer** [comment]
你好 请问能给我一份snow100k下载链接嘛 谷歌的失效了 麻烦您了

**Comment by sunshangquan** [comment]
@SeWngDer  
我用两个账号都试过了，还能打开啊，能看看是哪个链接失效了吗

**Comment by SeWngDer** [comment]
> [@SeWngDer](https://github.com/SeWngDer) 我用账号都试过了，还能打开两个啊，能看看哪个链接失效了吗

感谢您的回复，我想使用的数据集是50000对的snow100k 但是他们已经失效了 我在您别的回答中 找到了 snow100k-m 请问您还可以提供s和l嘛 麻烦了，也祝您心想事成

**Comment by sunshangquan** [comment]
@SeWngDer 是这个吗？我当时好像也只有这个：“Hi @Jerry-Jimmy , the test sets of Snow100K-S, -L and realistic are uploaded on [[Google Drive]](https://drive.google.com/drive/folders/1kLPWAev5f6OutKFALhxthuO1XCF5U1Yu?usp=drive_link) or [[Baidu disk]](https://pan.baidu.com/s/1oiL0IrB1vWo902G-jWk8LA?pwd=yuia) with pin: yuia.”

**Comment by SeWngDer** [comment]
> [@SeWngDer](https://github.com/SeWngDer)我当时的愿望也只有这个：“嗨@Jerry-Jimmy，Snow100K-S、-L和realistic的测试集已上传到[[Google Drive]](https://drive.google.com/drive/folders/1kLPWAev5f6OutKFALhxthuO1XCF5U1Yu?usp=drive_link)或[[百度盘]，](https://pan.baidu.com/s/1oiL0IrB1vWo902G-jWk8LA?pwd=yuia) pin：yuia。”

真是太感谢了，这么晚还麻烦您，不胜感激
